src/main/resources/static/templates/XXEVulnerability/LEVEL_1/XXE.js

```diff
@@ -74,8 +67,9 @@ function addingEventListener() {
   document.getElementById("addBook").addEventListener("click", load);
 }
 
+// Ensure that appendResponseCallback also uses textContent to prevent XSS
 function appendResponseCallback(data) {
-  document.getElementById("bookInformation").innerHTML = data;
```

Comment:
CWE-79: Improper Neutralization of Input During Web Page Generation ('Cross-site Scripting')
------
User controlled data in methods like `innerHTML`, `outerHTML` or `document.write` is an anti-pattern that can lead to XSS vulnerabilities